playbooks/acs.yml

```
@@ -406,7 +411,7 @@
         {{ lookup('url', downloads.adw_zip_sha1_checksum_url, username=nexus_user, password=nexus_password) }}
         {%- endif %}
       adf_app_version: "{{ adw.version }}"
-      adf_app_configuration: "{{ adf_app_oauth_configuration | default({}) }}"
+      adf_app_configuration: "{{ adf_app_oauth_configuration | default({}) | ansible.builtin.combine(adf_plugins_configuration) }}"
```

```suggestion
      adf_app_configuration: "{{ adf_plugins_configuration | ansible.builtin.combine(adf_app_oauth_configuration | default({}), recursive=true) }}"
```

@alxgomz please check

---

I can't think of cases where the two vars would overlap. And both are task level static vars, so both would work.